resources/specimin_command_cf-6019.txt

```diff
@@ -1 +1 @@
-./gradlew run --args='--root "../ISSUES/cf-6019/input/kafka-sensors/src/main/java/" --outputDirectory "../ISSUES/cf-6019/output" --targetFile "com/fillmore_labs/kafka/sensors/serde/confluent/interop/Avro2Confluent.java" --targetMethod "com.fillmore_labs.kafka.sensors.serde.confluent.interop.Avro2Confluent#transform(String, byte[])"'
\ No newline at end of file
+./gradlew run --args='--root "user/ISSUES/cf-6019/input/kafka-sensors/src/main/java/" --outputDirectory "user/ISSUES/cf-6019/output" --targetFile "com/fillmore_labs/kafka/sensors/serde/confluent/interop/Avro2Confluent.java" --targetMethod "com.fillmore_labs.kafka.sensors.serde.confluent.interop.Avro2Confluent#transform(String, byte[])"'
```

Comment:
Why do these need to change? The path here is still a relative path, so I don't see any benefit to changing this.